transformers/storage/utilities/wards/repository.go

```diff
@@ -49,7 +49,7 @@ func InsertWards(diffID, headerID int64, metadata storage.ValueMetadata, contrac
 	return tx.Commit()
 }
 
-func getUser(keys map[storage.Key]string) (string, error) {
+func GetUser(keys map[storage.Key]string) (string, error) {
```

Comment:
curious why we're exposing this method outside of this package now 🤔

Reply:
Nice catch! I was using it somewhere else, but ended up undoing the change, so this can totally be a private function.